swap_meet/item.py

```diff
@@ -1,2 +1,10 @@
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, category='', condition=0, age=0):
+        self.category = category
+        self.condition = condition
+        self.age = age
+    def __str__(self):
+        return "Hello World!"
+    def condition_description(self):
+        description_options = ['Poor','Used fair','Used good','Used very good','Used like new', 'New']
+        return description_options[self.condition]
```

Comment:
Excellent use of a data structure here to get the description based on condition - using a list enables you to avoid a long list of chained checks (if/elif/elif...) which can introduce bugs. 

There are still edge case behaviors we'd need to consider and handle: what if the condition is outside the expected range? what if the condition is a non-integer value?

A note about the variable description_options - since it's a constant you can indicate so by naming it with all caps `DESCRIPTION_OPTIONS` and even declare it outside the class function if you want the method to just have the logic of returning a condition.